Doc/library/dis.rst

```diff
@@ -1774,8 +1803,9 @@ These collections are provided for automatic introspection of bytecode
 instructions:
 
 .. versionchanged:: 3.12
-   The collections now contain pseudo instructions as well. These are
-   opcodes with values ``>= MIN_PSEUDO_OPCODE``.
+   The collections now contain pseudo instructions and instrumented
+   instructions as well. These are opcodes with values ``>= MIN_PSEUDO_OPCODE``
+   and ``>= MIN_INSTRUMENTED_OPCODE``.
```

Comment:
MIN_PSEUDO_OPCODE > MIN_INSTRUMENTED_OPCODE. 

Do we need to mention both?

Reply:
Because MIN_INSTRUMENTED_OPCODE is the smallest of the two, it has to be mentioned since it is the relevant one to access only the base opcodes. Previously it was not mentioned.

Reply:
so do we need to mention MIN_PSEUDO_OPCODE as well?

Reply:
Since the opcodes can be accessed in in the opcode module global, I am in favor of documenting the constants allowing to make sense of them yes.